markov/speech.py

```diff
@@ -76,7 +76,7 @@ def new_model(text):
     Cls = PosifiedText if settings.MODEL_LANG else markovify.NewlineText
     model = None
     try:
-        model = Cls(text, retain_original=False)
+        model = Cls(re.sub(r'["\']', '', text), retain_original=False)
```

Comment:
could you leave retain_original set as True, or set this variable from an environment variable?
it seems that the generated text is shortened when this param is False

Reply:
I think that setting this using an environment variable would be safer as it would be easier to change without touching the source code.